packages/blade/src/components/Typography/Code/Code.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Text } from '../Text';
+import { Code as CodeComponent } from './Code';
+
+const CodeStoryMeta: Meta = {
+  title: 'Components/Typography/Code',
+  component: CodeComponent,
+  args: {
+    size: 'medium',
+    type: 'subtle',
+    children: 'SENTRY_AUTH_TOKEN',
+  },
+};
+
+const CodeTemplate: ComponentStory<typeof CodeComponent> = (args) => (
+  // For React Native, use flex to align items correctly
```

Comment:
Didn't understand the context of this comment, seems unrelated 🤔

Reply:
As in that story will not look properly aligned on react native. For that we have to use flex and do it properly. I thought of conditionally adding a react native layout based on platform but might be overkill since storybook is mostly visited from web.